app/views/assessments/viewFeedback.html.erb

```diff
@@ -16,19 +16,26 @@ Graded by: <%= if @score.grader && @score.grader.user then @score.grader.user.em
 
 <% end %>
 <!--
-Yes, feedback if per problem but we show all annotations for this submission anyway. While this is not semantically correct, it offers a better UX than we had before. If we just show annotations attached to this problem, then where would we show annotations not attached to a problem at all?
+Yes, feedback is per problem but we show all annotations for this submission anyway. While this is not semantically correct, it offers a better UX than we had before. If we just show annotations attached to this problem, then where would we show annotations not attached to a problem at all?
 -->
 
 <% if @submission.annotations.count > 0 and @problemReleased %>
   <h2 style="margin-top: 1em">Remarks</h2>
 
   <ul>
+  <!-- TODO: Make order of annotations consistent with order of file list  -->
```

Comment:
Ah so this is a TODO for the future?

Reply:
Yup, it is a TODO to track the second "bug" reported by the 122 team (The order of annotations in the student feedback when they click on the score is not the same as the order when you look at the file page)